.github/workflows/ci-release.yml

```diff
@@ -0,0 +1,73 @@
+name: Publish release
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  publish-release:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: true
+
+    - name: Fetch git tags
+      run: |
+        git fetch --prune --unshallow --tags
+
+    - uses: actions/setup-go@v2
+      with:
+        go-version: ^1.15
+
+    - uses: actions/setup-node@v2-beta
+      with:
+        node-version: '10'
+
+    - uses: docker/login-action@v1
+      id: dockerhub-login
+      with:
+        username: jaegertracingbot
+        password: ${{ secrets.DOCKERHUB_TOKEN }}
+      env:
+        DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
+      if: env.DOCKERHUB_TOKEN != null
+
+    - name: Export DOCKERHUB_LOGIN variable
+      run: |
+        echo "DOCKERHUB_LOGIN=true" >> $GITHUB_ENV
+      if: steps.dockerhub-login.outcome == 'success'
+
+    - name: Export BRANCH variable
+      uses: ./.github/actions/setup-branch
+
+    - name: Install tools
+      run: make install-ci
+
+    - name: Build binaries
+      run: make build-all-platforms
+
+    - name: Package binaries
+      run: bash scripts/travis/package-deploy.sh
+      env:
+        DEPLOY: true
+
+    - name: Upload binaries
+      uses: svenstaro/upload-release-action@v2
+      with:
+        file: deploy/*.tar.gz
+        file_glob: true
+        tag: ${{ github.ref }}
+        repo_token: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Build docker images
+      run: make docker
+      env:
+        DOCKER_NAMESPACE: jaegertracing
```

Comment:
not needed

https://github.com/jaegertracing/jaeger/blob/2ad44d581c71b8cfb66b0c7ef153a0fd5dcdb6db/Makefile#L69